qhub/template/image/Dockerfile.jupyterhub

```diff
@@ -24,23 +17,7 @@ RUN /opt/scripts/install-conda.sh
 # ========== jupyterhub install ===========
 COPY jupyterhub/environment.yaml /opt/jupyterhub/environment.yaml
 COPY scripts/install-conda-environment.sh /opt/scripts/install-conda-environment.sh
-RUN /opt/scripts/install-conda-environment.sh /opt/jupyterhub/environment.yaml 'false'
-
-COPY jupyterhub /opt/jupyterhub
-RUN /opt/jupyterhub/postBuild
-
-# ========= Create USER =============
-RUN adduser --disabled-password \
-    --gecos "Default user" \
-    --uid ${NB_UID} \
-    --home ${HOME} \
-    --force-badname \
-    ${NB_USER}
-
-WORKDIR /srv/jupyterhub
-
-# So we can actually write a db file here
-RUN fix-permissions /srv/jupyterhub
```

Comment:
This looks like the line that is causing issues with jupyterhub. I think I was a little too ambitious asking we remove fix-permissions as well. Lets make this PR only remove the jovyan user. And keep all the other docker images as is. Totally agree that this would be worth fixing later. 

The proper way to fix this would be to use https://kubernetes.io/docs/concepts/storage/volumes/#emptydir with the kubernetes deployment. Not important for now. Lets just simplify this PR to have a smaller scope.

Reply:
Thanks for the feedback @costrouc! I'll add the `fix-permissions` back and test.